lib/action_subscriber/dsl.rb

```diff
@@ -1,5 +1,54 @@
+
 module ActionSubscriber
   module DSL
+    class Filter
+      attr_accessor :method
+      attr_accessor :included_actions
+      attr_accessor :excluded_actions
+
+      def initialize(method, options)
+        @method = method
+        @included_actions = @excluded_actions = []
+        parse_options(options)
+      end
+
+      def matches(action)
```

Comment:
Is it worth memoizing the matching action here? Seems like a safe thing to do. Not sure if it's necessary, though.

Reply:
Not quite sure what you mean.  `action` will change with each iteration, so we'd have to memoize for each action.  I don't see the value, unless I'm missing something?

Reply:
Yup, memoize on action. I don't think it's necessary now to optimize, because I don't think the gains will be massive, but just leaving a note.